numba/testing.py

```diff
@@ -1,9 +1,522 @@
 from __future__ import print_function, division, absolute_import
 
 import sys
+import os
 import functools
-
+import numba.unittest_support as unittest
+from unittest import result, runner, signals, suite, loader, case
+from numba.utils import PYVERSION, StringIO
 from numba import config
+import argparse
+import collections
+import contextlib
+import cProfile
+import multiprocessing
+from fnmatch import fnmatch
+from os.path import dirname, join, abspath, relpath
+
+try:
+    from multiprocessing import TimeoutError
+except ImportError:
+    from Queue import Empty as TimeoutError
+
+
+class TestLister(object):
+    """Simply list available tests rather than running them."""
+    def __init__(self):
+        pass
+
+    def run(self, test):
+        result = runner.TextTestResult(sys.stderr, True, 1)
```

Comment:
Just a nit, but `True` and `1` are not very explicit arguments if you don't name them ;-)